cmake/modules/FindPyArrow.cmake

```diff
@@ -50,17 +50,20 @@ set(PYTHON_PYARROW_LIBRARY_DIR ${__pyarrow_library_dirs})
 
 if (CMAKE_SYSTEM_NAME MATCHES "Darwin")
   # Link against pre-built libarrow on MacOS
-  set(PYTHON_PYARROW_SHARED_LIBRARY ${__pyarrow_library_dirs}/${CMAKE_SHARED_LIBRARY_PREFIX}arrow.15${CMAKE_SHARED_LIBRARY_SUFFIX})
+  set(PYTHON_PYARROW_PYTHON_SHARED_LIBRARY ${__pyarrow_library_dirs}/${CMAKE_SHARED_LIBRARY_PREFIX}arrow_python.15.dylib)
+  set(PYTHON_PYARROW_ARROW_SHARED_LIBRARY ${__pyarrow_library_dirs}/${CMAKE_SHARED_LIBRARY_PREFIX}arrow.15.dylib)
+else()
+  # linux
+  set(PYTHON_PYARROW_PYTHON_SHARED_LIBRARY ${__pyarrow_library_dirs}/${CMAKE_SHARED_LIBRARY_PREFIX}arrow_python${CMAKE_SHARED_LIBRARY_SUFFIX}.15)
+  set(PYTHON_PYARROW_ARROW_SHARED_LIBRARY ${__pyarrow_library_dirs}/${CMAKE_SHARED_LIBRARY_PREFIX}arrow${CMAKE_SHARED_LIBRARY_SUFFIX}.15)
```

Comment:
This step fails on the docker image for python3, as far as I can discern.

```bash
CMake Error at /opt/cmake/share/cmake-3.15/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find PyArrow: Found unsuitable version "", but required is at
  least "0.15" (found PYTHON_PYARROW_INCLUDE_DIR-NOTFOUND)
```

Reply:
did you rebuild the docker image? running from scratch this works fine for me, the error you're getting suggest pyarrow was not installed